Source/Core/DiscIO/WIABlob.cpp

```diff
@@ -653,7 +653,11 @@ bool WIARVZFileReader<RVZ>::Chunk::Read(u64 offset, u64 size, u8* out_ptr)
     return false;
   }
 
-  while (offset + size > m_out.bytes_written - m_out_bytes_used_for_exceptions)
+  const auto out_bytes_written_excluding_exceptions = [this] {
```

Comment:
Maybe `get_out_bytes_written_excluding_exceptions` to make it clearer it's a lambda and not an integer?

Reply:
I've turned it into a member function `GetOutBytesWrittenExcludingExceptions` instead.